server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/resources/cucumber/DownloadGet.feature

```diff
@@ -23,80 +23,80 @@ Feature: Download GET
     And the user ask for messages "m1"
-    When "username@domain.tld" downloads the message by its blobId
+    When "alice@domain.tld" downloads the message by its blobId
     Then the user should receive that blob
     And the blob size is 36
 
   Scenario: Deleted message should revoke attachment blob download rights
-    Given "username@domain.tld" mailbox "INBOX" contains a message "1" with an attachment "2"
-    And "username@domain.tld" delete mailbox "INBOX"
-    When "username@domain.tld" downloads "2"
+    Given "alice@domain.tld" mailbox "INBOX" contains a message "1" with an attachment "2"
+    And "alice@domain.tld" delete mailbox "INBOX"
+    When "alice@domain.tld" downloads "2"
     Then the user should receive a not found response
 
   Scenario: User cannot download attachment of another user
-    Given "username@domain.tld" mailbox "INBOX" contains a message "1" with an attachment "2"
-    And a connected user "username1@domain.tld"
-    And "username1@domain.tld" has a mailbox "INBOX"
-    When "username1@domain.tld" downloads "2"
+    Given "alice@domain.tld" mailbox "INBOX" contains a message "1" with an attachment "2"
+    And a connected user "bob@domain.tld"
+    And "bob@domain.tld" has a mailbox "INBOX"
+    When "bob@domain.tld" downloads "2"
     Then the user should receive a not found response
 
   Scenario: User cannot download message blob of another user
-    Given "username@domain.tld" mailbox "INBOX" contains a message "1" with an attachment "2"
-    And a connected user "username1@domain.tld"
-    And "username1@domain.tld" has a mailbox "INBOX"
-    When "username1@domain.tld" downloads "1"
+    Given "alice@domain.tld" mailbox "INBOX" contains a message "1" with an attachment "2"
+    And a connected user "bob@domain.tld"
+    And "bob@domain.tld" has a mailbox "INBOX"
+    When "bob@domain.tld" downloads "1"
     Then the user should receive a not found response
 
-  Scenario: User cannot download attachment of another user when shared mailbox
-    Given "username@domain.tld" mailbox "sharedMailbox" contains a message "1" with an attachment "2"
-    And "username@domain.tld" shares its mailbox "sharedMailbox" with "username1@domain.tld"
-    And a connected user "username1@domain.tld"
-    And "username1@domain.tld" has a mailbox "sharedMailbox"
-    When "username1@domain.tld" downloads "2"
+  Scenario: User can download attachment of another user when shared mailbox
+    Given "alice@domain.tld" mailbox "sharedMailbox" contains a message "1" with an attachment "2"
+    And "alice@domain.tld" shares its mailbox "sharedMailbox" with "bob@domain.tld"
+    And a connected user "bob@domain.tld"
+    And "bob@domain.tld" has a mailbox "sharedMailbox"
+    When "bob@domain.tld" downloads "2"
     Then the user should receive that blob
     And the blob size is 3071
 
-  Scenario: User cannot download message blob of another user when shared mailbox
```

Comment:
this s/can/cannot should not be done in this commit 8443e97001d50990196737a399471773e9041165

Reply:
Done

Reply:
Done